setup/Swix/Microsoft.FSharp.Compiler/Files.swr

```diff
@@ -0,0 +1,34 @@
+use vs
+
+package name=Microsoft.FSharp.Compiler
+        version=$(FSharpPackageVersion)
+
+folder "InstallDir:MSBuild\15.0\Bin\FSharp"
+  file source="$(BinariesFolder)\net40\bin\fsc.exe" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\fsc.exe.config"
+  file source="$(BinariesFolder)\net40\bin\FSharp.Build.dll" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\FSharp.Compiler.Interactive.Settings.dll" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\FSharp.Compiler.Private.dll" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\FSharp.Compiler.Server.Shared.dll" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\FSharp.Core.dll" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\FSharp.Core.optdata"
+  file source="$(BinariesFolder)\net40\bin\FSharp.Core.sigdata"
+  file source="$(PackagesFolder)\Microsoft.VisualFSharp.Type.Providers.Redist.$(MicrosoftVisualFSharpTypeProvidersRedistPackageVersion)\content\4.3.0.0\FSharp.Data.TypeProviders.dll" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\fsi.exe" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\fsi.exe.config"
+  file source="$(BinariesFolder)\net40\bin\fsiAnyCpu.exe" vs.file.ngen=yes
+  file source="$(BinariesFolder)\net40\bin\fsiAnyCpu.exe.config"
+  file source="$(BinariesFolder)\net40\bin\Microsoft.Build.Conversion.Core.dll"
+  file source="$(BinariesFolder)\net40\bin\Microsoft.Build.dll"
+  file source="$(BinariesFolder)\net40\bin\Microsoft.Build.Engine.dll"
```

Comment:
Are you redistributing MSBuild assemblies?

Reply:
Yes.  Our current MSI ships these, too.  If you have F# installed on your machine these assemblies live next to our compiler at `C:\Program Files (x86)\Microsoft SDKs\F#\10.1\Framework\v4.0`.